globus_action_provider_tools/flask/helpers.py

```diff
@@ -130,13 +131,17 @@ def blueprint_error_handler(exc: Exception) -> ViewReturn:
 
 
 def validate_input(
-    request_json: Dict[str, Any], input_body_validator: ActionInputValidatorType
+    request_json: Any, input_body_validator: ActionInputValidatorType
 ) -> ActionRequest:
     """
-    Ensures the incoming request conforms to the ActionRequest schema and
-    the user-defined ActionProvider input schema.
+    Verify the incoming request is a JSON object that conforms to the required schemata.
```

Comment:
I was today years old when I learned that the plural for `schema` was `schemata`...